millisom/GluGuide
# Like / Dislike for Posts and comments

# What changed?
- added like functionality for posts
- added like / dislike functionality for comments
- improved code styling

I did not add any styling yet, as I am waiting for the pull request for the css update. I can either update the styling in this Pull Request or later.

# How to test?

* Create a post or visit an existing one.
* Look for the new Like Button and press it.
* Reload the page, your like should still be visible. Remove it and do the same, the like should still stay.
* View the post on the start page and in "Your Blog", you should see the amount of likes the post has.
* Now add a comment to any post and look for the new "Like" "Dislike" Button. 
* You should be able to like a comment or dislike. Not both. And it should stay the same after a reload.
